capabilities/working-in-small-batches.md

```diff
@@ -1,52 +1,69 @@
 
 ## Nuances
 
 This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this capability. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the capability with your teams.
 
-### Managing Organizational Resistance
+### Thoughtful, Collaborative Upfront Design
+
+Traditionally, software requirements are conceived by leaders or product specialists. The focus is usually on new features or functionality, covering all bells and whistles and all known edge cases. The ideas are presented to designers and developers who must then turn these dreams into reality. But dependencies, intertwined logic, and incorrect assumptions tend to surface during implementation. This workflow is not conducive to working in small batches. 
+
+Breaking work into small, independently deliverable batches requires thoughtful upfront software design. Product, engineering, and design teams need to collaborate on requirements early and be willing to continually rethink designs and priorities throughout implementation.
+
+### Misinterpretation of "Small Batches"
+
+Teams sometimes mistake “small batches” for “small tasks” without considering the value each batch delivers. If small batches lack user value, or fail to provide meaningful feedback, then they don't support the goal of rapid iteration.
 
-Traditionally, software requirements are conceived by either leaders in the organization or with product specialists. Usually the focus is on new features or functionality. When people design systems, they want them to be great. So it's common for them to be conceived with all of the bells and whistles, covering all known edge cases. Then the ideas are presented to designers and developers to turn those dreams into reality. This workflow is not conducive to working in small batches.
+### Neglecting Long-Term Objectives
 
-Changes tend to have hidden dependencies, intertwined logic, and assumptions that only become clear during implementation. Breaking work into genuinely small, independently deliverable batches demands thoughtful upfront design. Product, engineering, and design teams need to collaborate and have a willingness to rethink designs and priorities continually.
+A narrow focus on small, immediate "tasks" can distract from the project's long-term objectives. It's important to balance the benefits of incremental work with broader strategic planning to ensure that all updates represent *meaningful contributions* to the project's overarching goals.
 
-### Misinterpretation of Small Batches
+## Assessment
+To assess how mature your team or organization is in this capability, complete this short exercise.
```

Comment:
My markdown formatter spits out warnings when text immediately follows a heading. Can you add a newline between lines 27 and 28? I've been doing that for all of our other files.